### PR TITLE
small slew of op modal bugfixes

### PR DESCRIPTION
**Pull request by jacobheric**

### CODE REVIEW

File: islands/opModal.tsx [hunk 1]
```diff
@@ -44,12 +41,6 @@ export default function OpModal(
   const clients = opModal.value?.clients;
   const opType = OperationType[audience?.operationType];
 
-  useEffect(() => {
```
Comment (jacobheric) [inline islands/opModal.tsx:47]: Was stomping non audience data on operation change.

**Review by Adelle-Pitsas — APPROVED** [review]

File: islands/opModal.tsx [hunk 2]
```diff
@@ -68,7 +59,7 @@ export default function OpModal(
         schema: JSON.stringify(schema.schema, null, 2),
       };
     }
-  }, [audience]);
+  }, [audience, schemaNavOpen]);
```
Comment (jacobheric) [inline islands/opModal.tsx:62]: Make the fetch lazy rather than eager to lighten the fetch load a bit.

File: islands/opModal.tsx [hunk 3]
```diff
@@ -153,8 +144,8 @@ export default function OpModal(
                               {audience?.operationName}
                             </h3>
                             <p class="text-xs text-cloud">
-                              {`${clients.length} attached client${
-                                (clients.length !== 1) ? "s" : ""
+                              {`${clients?.length || 0} attached client${
```
Comment (jacobheric) [inline islands/opModal.tsx:147]: Was blowing up when no clients where attached.

File: islands/peek.tsx [hunk 4]
```diff
@@ -70,7 +70,7 @@ export const PeekRow = (
         <pre>
           <code>
             <div dangerouslySetInnerHTML={{
-                __html: hljs.highlightAuto(parseData(row.newData)).value,
+                __html: hljs.highlightAuto(parseData(row.newData && row.newData.length > 0 ? row.newData : row.originalData)).value,
```
Comment (jacobheric) [inline islands/peek.tsx:73]: Make sure there is actually data and not an empty array.

File: lib/nodeMapper.ts [hunk 5]
```diff
@@ -192,7 +192,7 @@ export const mapNodes = (
         dragHandle: `#${cKey}-dragHandle`,
         position: {
           x: offset(cKey, nodesMap.components),
-          y: 650 + (max - 1) * 76,
```
Comment (jacobheric) [inline lib/nodeMapper.ts:195]: Remove the lower throughput edge and pull the data source widgets back up a bit.

**Review by Adelle-Pitsas — COMMENTED** [review]
Awesome! Thanks for cleaning these up

**Pull request merged**